_posts/2016-09-01-sept-all-stars.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+subheadline: "Community"
+title: "September Data Carpentry All-Stars!"
+teaser: "Recognizing outstanding work from our volunteer community."
```

Comment:
It seems this is to call out _volunteers_ for recognition, _because_ of their work, rather than calling out the work. So I would change this to say that it is about recognizing outstanding volunteers from the community.